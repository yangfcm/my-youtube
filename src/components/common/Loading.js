import React from "react";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";

const Loading = () => {
  return (
    <div className="text-center my-3 py-3">
      <FontAwesomeIcon icon="spinner" className="mx-2" />
      Loading...
    </div>
  );
};

export default Loading;
