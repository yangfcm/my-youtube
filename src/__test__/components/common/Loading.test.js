import React from "react";
import { shallow } from "enzyme";
import Loading from "components/common/Loading";

describe("Test Loading component", () => {
  it("should render Loading component correctly", () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
