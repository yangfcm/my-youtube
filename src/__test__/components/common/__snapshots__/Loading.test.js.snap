// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loading component should match snapshot 1`] = `
<div
  className="text-center my-3 py-3"
>
  <div
    className="spinner-border text-center"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
  <div>
    Loading...
  </div>
</div>
`;
